### PR TITLE
Update Customization Guide

### PR DESCRIPTION
**Pull request by shahx95**
#### Describe your changes
Created a customization checklist that lists down the changes any user would have to make if they plan to use the template for their own event. This checklist guide contains changes that need to be made in the nav bar, the event details section, the talks section, and the bottom bar. Each change also carries a brief instruction on how to implement the change.

#### Link to issue this resolves
https://github.com/brittanyjoiner15/eui-event-template/issues/21

#### Screenshot of changes(if relevant)

* [x] The **title** should be something descriptive about what you're changes do. _(It will default to whatever you put as your commit message.)_
* [x] This **description** should include ...
  * [x] A short summary of your changes
  * [x] A link to the issue that relates to these changes
  * [ ] A screenshot of the changes if possible!
* [ ] Add `brittanyjoiner15` as a reviewer
* [x] Add yourself as assignee

### CODE REVIEW

File: docs/customization_checklist.md [hunk 1]
```diff
@@ -1 +1,40 @@
-Want to use this for your own event? Here's what you need to change:
+## Welcome to EUI Event Template customization guide 
+
+If you want to use this template for your own event, here is everything that you need to change along with a brief description of how to make the respective change. The guide has been divided by sections of the web app:
+
+### Nav Bar:
+
+- [ ] **Change logo and favicon:** Follow the path ```eui-event-template/src/data/consts.js```. In the ```const.js``` file, on _line 12_, replace the placeholder rainbow cluster link with a link of your event’s logo.
```
Comment (brittanyjoiner15) [inline docs/customization_checklist.md]: What do you think about leaving a permalink here to reference that line? Reference this link: 
https://github.com/brittanyjoiner15/eui-event-template/blob/8cefe4c73b1dea7f5fa33c6bd15071cb1bbae463/src/data/consts.js#L12

File: docs/customization_checklist.md [hunk 2]
```diff
@@ -1 +1,40 @@
...
+
+- [ ] **Change speaker avatar:** Follow the path ```eui-event-template/src/images/```. Delete placeholder avatars. Add the avatars of your actual speakers to the images folder. Then follow the path ```eui-event-template/src/data/speakers.js```. In the ```speakers.js``` file, import the avatars of the respective speakers from the images folder and replace the placeholder avatar in the speaker objects with your actual avatars.
+
+- [ ] **Change speaker layout:** This is an optional change. Depending on the number of speakers in your event, you may want to change the layout of the speakers section. Follow the path ```eui-event-template/src/components/panels/SpeakersPanel.jsx```. In the ```SpeakersPanel.jsx``` file, replace the value of columns in _line 52_. The current placeholder value is 3, which is why the speaker section renders with 3 columns.
```
Comment (brittanyjoiner15) [inline docs/customization_checklist.md]: This is a good point! Great job including that!

File: docs/customization_checklist.md [hunk 3]
```diff
@@ -1 +1,40 @@
...
+
+- [ ] **Change Add to Calendar link:** Follow the path ```eui-event-template/src/data/consts.js```. In the ```const.js``` file, _within the sessionOne and sessionTwo objects_, there will be calendarLink attributes. Replace the placeholder with your Add to Calendar link. 
+
+### Event Details:
+
+- [ ] **Replace hero image:** Follow the path ```eui-event-template/src/components/panels/EventDetails.jsx```. In the ```EventDetails.jsx``` file, _within the renderHero function (lines 13 - 21)_, replace the existing URL in the src attribute with the URL of your hero image. Replace the alt attribute’s text with text appropriate for your hero image. 
+
+- [ ] **Replace hero title and subtitle:** Follow the path ```eui-event-template/src/data/speakers.js```. In the ```speakers.js``` file, _within the speakers array (line 8)_, each speaker is listed as an object. Overwrite the placeholder information with your the actual information of your event’s speakers. If you want to add more speakers, add an additional object to the array. If you want to decrease the placeholder speakers, just remove the objects from the speakers array. 
+
```
Comment (brittanyjoiner15) [inline docs/customization_checklist.md:20]: Might want to add another `### Speaker` header here like you did for Event Details and Talks sections

File: docs/customization_checklist.md [hunk 4]
```diff
@@ -1 +1,40 @@
...
+- [ ] **Change speaker layout:** This is an optional change. Depending on the number of speakers in your event, you may want to change the layout of the speakers section. Follow the path ```eui-event-template/src/components/panels/SpeakersPanel.jsx```. In the ```SpeakersPanel.jsx``` file, replace the value of columns in _line 52_. The current placeholder value is 3, which is why the speaker section renders with 3 columns.  
+
+### Talks:
+
+- [ ] **Change schedule**: Follow the path ```eui-event-template/src/data/talks.js```. In the ```talks.js``` file, there is a _talks array (line 7)_ holding each individual talk as an object. Each object has attributes. You can start by overwriting the placeholder attributes. If you need to add more talks, add objects to the _talks_ array. If you need to remove talks, just delete the respective objects from the _talks_ array. Do note that the time in the sessionTime attribute is in GMT. **All attribute formats must be maintained**.
```
Comment (brittanyjoiner15) [inline docs/customization_checklist.md]: Good callout about the GMT time!

**Review by brittanyjoiner15 — CHANGES_REQUESTED** [review]
Overall this is awesome and really close! Added a couple comments, and when you reference things like _line 12_, let's change that for now because those line numbers could change. Instad if we can reference a permalink to the code that would be great. Like this
---

- [ ] ** Change logo text:** Go to the [`rainbowLetters.jsx` file](https://github.com/brittanyjoiner15/eui-event-template/blob/8cefe4c73b1dea7f5fa33c6bd15071cb1bbae463/src/utilities/rainbowLetters.jsx#L27) and replace `"{rainbowedLetters} SUMMIT" with the name of your event.

----
If you can make those changes to the places where you reference lines, i think that would be cleaner and easier to reference! But overall this looks great!

**Comment by brittanyjoiner15** [comment]
@shahx95 hey just wanted to check in on this and see if you had a chance to make the requested changes?

**Comment by shahx95** [comment]
> @shahx95 hey just wanted to check in on this and see if you had a chance to make the requested changes?

Hi @brittanyjoiner15. I'll be on it today

**Comment by shahx95** [comment]
Hey @brittanyjoiner15. I have made the requested changes 😊

**Review by brittanyjoiner15 — APPROVED** [review]
This all looks great! Thanks for making those changes!

**Pull request merged**